src/Tree/index.js

```diff
@@ -82,6 +82,13 @@ export default class Tree extends React.Component {
           .scaleExtent([scaleExtent.min, scaleExtent.max])
           .on('zoom', () => {
             g.attr('transform', `translate(${event.translate}) scale(${event.scale})`);
+            if (typeof onUpdate !== 'undefined') {
```

Comment:
Unless there's a specific reason for the difference I'd say let's keep this consistent with line 198; i.e. also make it `typeof onUpdate === 'function'`

Reply:
Fixed. This is a matter avoiding an error, not just consistency. 😄